test/types/enum/vass/enum-to-uint-1.chpl

```diff
@@ -5,17 +5,17 @@ config var e1var: e1;
 config var e2var: e2;
 
```

Comment:
With this change, this test checks explicit casting from simple enums to uints. If we have such checks elsewhere, this test can be removed. Otherwise it looks good.